src/main/java/com/findmyrecycling/fmrenterprise/FindMyRecyclingController.java

```diff
@@ -30,7 +29,7 @@ public String index(Model model) {
         return "index";
     }
 
-    @RequestMapping("/AddFacility.html")
+    @RequestMapping("/addFacility.html")
```

Comment:
Names in java, even in the annotation of a spring endpoint, should be camel case. I would recommend changing this to keep your naming convention consistent.

Source: https://www.baeldung.com/java-clean-code

Reply:
URLs are often completely lowercase... except for .aspx files.  :)  I might consider making this all lowercase.